plugins/holiday-calculator/build.gradle

```
@@ -0,0 +1,8 @@
+dependencies {
+	implementation 'org.itsallcode:holiday-calculator:0.1.0'
+
+    runtimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}") {
```

Should only be required for tests?
```suggestion
    testRuntimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}") {
```

---

I'm not sure. 
In main-runtime logging seems fine - maybe a logger is provided from somewhere else? 
If so, then yes, you're right: only in test I noticed a logger to seem to have been missing.